frontend/lib/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -45,69 +45,72 @@ export function GraphVizChart({
 }: GraphVizChartProps): ReactElement {
   const chartId = `graphviz-chart-${element.elementId}`
 
+  const isFullScreen = Boolean(propHeight)
+  const isFullScreenRef = useRef(isFullScreen)
+  // Update isFullScreen state whenever propHeight changes
+  useEffect(() => {
+    isFullScreenRef.current = isFullScreen
+  }, [isFullScreen])
```

Comment:
Hey Ken! Thanks for investigating this issue. Your PR works as expected and is a whole lot simpler since it uses the `isFullScreen` prop passed down by `withFullScreenWrapper` 😄 Should I close this one?

Reply:
Hey @snehankekre ! Glad it ended up being a bit easier. Would you mind taking the code and bringing it over the finish line? Looks like you got the E2E tests, seems like we can merge our code and ensure it's well-tested?